tests/README.md

```diff
@@ -43,3 +43,23 @@ _Note: You can run `trezor-user-env` as standalone process with graphical output
 1. Create or modify file in `tests/__fixtures__`
 1. Make sure it is imported in `tests/__fixtures__/index.js`
 1. Make sure the method you are testing is listed in `.github/workflows/*.yml` and `.gitlab-ci.yml`
+
+## Transactions cache
+Bitcoin-like coins `signTransaction` method require additional data about transactions referenced from used inputs.
+
+Those data are automatically downloaded using backed defined in `coins.json` by default if `refTxs` param is not specified.
```

Comment:
I guess...?

```suggestion
Those data are automatically downloaded from backend defined in `coins.json` by default if `refTxs` param is not specified.
```

Reply:
317be40

Reply:
The patch didn't fix the typo in `backed` (should be `backend` I believe).

Not important at all -- feel free to reslove this.

Reply:
ah, spell checker didn't catch this :) `backed` is also a word :)
thanks